include/sway/tree/container.h

```diff
@@ -212,4 +218,9 @@ void container_calculate_title_height(struct sway_container *container);
 
 void container_notify_child_title_changed(struct sway_container *container);
 
+/**
+ * Return the height of a regular title bar.
+ */
+size_t container_titlebar_height();
```

Comment:
Very small point, but you should put an explicit `(void)` as the arguments for a prototype.
An empty argument list for prototypes has surprising results in C.

Reply:
Ah yeah, you or someone else has mentioned that to me before. I'll change it.

Reply:
I may as well give the reason now, in case anyone stumbles on this comment again.
An empty argument list in a prototype is actually declaring a function with _unspecified_ arguments, not no arguments.
So if you tried to call it like `container_titlebar_height(100);`, you wouldn't get a warning.